media/pages/sk/naco-infozakon/page.html

```diff
@@ -60,6 +60,7 @@ <h2 id="kauza-predrazene-ct-v-piestanskej-nemocnici">Kauza Predražené CT v pie
 zmluvu a ušetriť nemocnici 600 000 eur.
 </p>
 
+<hr>
```

Comment:
@martinmacko47 Vyuzijeme uz existujuci style  `<hr>` **1px seda linka #eee** tagu alebo podla zadania: ***2px seda linka #cccccc**?

 Ak podla zadania, mam vytvorit jednu classu, ktora definuje presne tento styl, alebo napisem k tomu mixin, kde bude mozne nastavit hrubku (a mozno aj farbu) ciary?

Reply:
Okrem zdrojov `<hr>` pouzivame iba na jednom mieste. Na oddelenie "Použiť iný login" na homepage. IMHO oba tieto `<hr>` by mali byt rovnake. Ak teda `<hr>` zmenime, tak vsade rovnako.

Co sa tyka konkretnej hrubky a farby. Cakal by som, ze hrubka aj farba `<hr>` budu rovnake ako hrubka a farba borderu v pravom menu. A to by som cakal, ze bude rovnake ako border v boxoch pre blogy/casestudies. Ak to tak nie je, zislo by sa to vsade zjednotit.

Reply:
:ok: Pouzil som farbu, ktora je na blog/casestudy `chv-color-gray-ll`